src/pages/Main/components/Technologies/Technologies.module.scss

```diff
@@ -13,14 +29,47 @@
         width: 32.7%;
         max-height: 24.0625rem;
         margin: 0.375rem 0;
+        overflow: hidden;
+        transition: .3s;
+        &_Big {
+            display: flex;
+            position: relative;
+            width: 49.523%;
+            max-height: 24.0625rem;
+            margin: 0.375rem 0;
+            overflow: hidden;
+            transition: .3s; 
+            &:hover {
+                .technoPic__title {
+                    right: 0;
+                    transition: .5s;
+                }
+            }              
+        }
+        &:hover {
+            .technoPic__title {
+                right: 0;
+                transition: .5s;
+            }
```

Comment:
please add hover to .technoPic__title for screen resolution under 768px

Reply:
What do you mean? Why? It's everywere is that is need